tokenizer/test4.test

```diff
@@ -438,8 +438,8 @@
     { "code": "invalid-character-sequence-after-doctype-name", "line": 1, "col": 16 }
 ]},
 
-{"description":"Doctype sistem",
-"input":"<!DOCTYPE html SISTEM \"AbC\">",
```

Comment:
Do we want to change the actual test here? It's probably not a bad idea to keep testing typos.

Reply:
And it looks like it's specifically trying to test this behaviour with this typo?

Reply:
i was wondering that. I can revert these if so.

Reply:
Please revert changes to test inputs.